views/explorer.tmpl

```diff
@@ -22,14 +22,20 @@
 
         <div class="row">
             <div class="col-md-12">
-                <table class="table striped">
+                <table class="table striped table-responsive">
                     <thead>
                         <tr>
                             <th>Height</th>
-                            <th>Votes</th>
-                            <th>Tickets</th>
-                            <th>Transactions</th>
-                            <th>Revocations</th>
+                            <th>
+                                <span class="d-none d-md-inline">Transactions</span>
+                                <span class="d-md-none">Txn</span>
+                            </th>
+                            <th>Vote<span class="d-none d-md-inline">s</span></th>
+                            <th>Ticket<span class="d-none d-md-inline">s</span></th>
+                            <th>
+                                <span class="d-none d-md-inline">Revocations</span>
+                                <span class="d-md-none">Revoke</span>
```

Comment:
I tried just "Rev" and it gets the whole table on the phone without scrolling.  It's not a bad abbreviation considering we have Txns for Transactions.  :)